tests/beman/inplace_vector/spec.test2.experimental.cpp

```diff
@@ -0,0 +1,1360 @@
+#include "beman/inplace_vector/inplace_vector.hpp"
+#include "gtest/gtest.h"
+#include <algorithm>
+#include <concepts>
+#include <iterator>
+#include <tuple>
+#include <type_traits>
+
+namespace {
+
+// We run the tests on various element types with the help of GoogleTest's
+// typed tests. Which types shall we use?
+// We know from the specification of inplace_vector that there are separate
+// implementation branches for trivial and non-trivial types (due to constexpr
+// support). Furthermore, we have the requirements that the triviality of
+// inplace_vector's SMFs depends on the triviality of the element type T:
```

Comment:
I hope this is not a stupid question. 
What is SMF?

Reply:
Thanks for asking. I'm using this all the time so I didn't even notice it's not obvious.
SMF = special member function, i.e. default constructor, copy constructor, move constructor, destructor, copy assignment, move assignment.

Reply:
Thx for the clarification 👌